lib/mixlib/shellout/version.rb

```diff
@@ -1,5 +1,5 @@
 module Mixlib
   class ShellOut
-    VERSION = "2.4.4".freeze
+    VERSION = "3.0.0".freeze
```

Comment:
This is technically unnecessary. It doesn't hurt to do it, but the `bash:.expeditor/update_version.sh` action would update this with `3.0.0`.